news/666.update.rst

```diff
@@ -0,0 +1 @@
+Modernize the torch for ``hook`` and reduce the amount of unnecessarily collected data files (header files and static libraries). Requires PyInstaller >= 6.0.
```

Comment:
That was supposed to be "hook for torch" (and was a mistake that I made in the comment).